### PR TITLE
client: Read latLong search param and enforce region in list and map

### PR DESCRIPTION
**Pull request by mikol**
Wildfire Check 2022-09-06 15-17-21

### CODE REVIEW

File: src/v2/modules/isLineWithinLatitudes.mjs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  const deltaVertical = long2 - long1
+
+  if (Math.abs(deltaVertical) < 0.01) {
+    // Too close to vertical.
+    return false
+  }
+
+  // Parameterized value on line segment where 0 represents `[lat1, long1]` and
+  // 1 represents `[lat2, long2]`.
+  const point = (longitude - long2) / deltaVertical
```
Comment (kinshuk) [inline src/v2/modules/isLineWithinLatitudes.mjs:41]: shouldn't that be longitude - long1?  Maybe update testcases.

Comment (mikol) [reply src/v2/modules/isLineWithinLatitudes.mjs:41]: I’ll take a look

File: src/v2/components/PotentialFireList.jsx [hunk 2]
```diff
@@ -96,7 +104,7 @@ export default function PotentialFireList(props) {
         // Fire is already indexed, but its video has been updated.
         firesByKey[key].croppedUrl = croppedUrl
         // TODO: Update video source, reload video.
-        console.error('Not implemented: updateFires()')
+        ['console'].error('Not implemented: updateFires()')
```
[the remark this answers is not in the document]
Comment (mikol) [reply src/v2/components/PotentialFireList.jsx:107]: This is there temporarily until I finish the implementation, which is why I distinguished it from `report`

File: src/v2/modules/isLineWithinLatitudes.mjs (hunk 1 above)
Comment (mikol) [reply src/v2/modules/isLineWithinLatitudes.mjs]: Fixed

**Review by kinshuk — APPROVED** [review]

**Pull request merged**